contracts/stellar/context-config/deploy_devnet.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+set -e
+
+# Get the directory where the script is located
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
is there anything in this script that needs the CWD where it's called from?

otherwise, you can skip saving `SCRIPT_DIR` if you just did

```
cd "$(dirname $0)"
```

at the start of this file